src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -124,14 +165,19 @@ private static void init(){}
         public static final IOptionalNamedTag<Block> STAINED_GLASS_PANES = tag("stained_glass_panes");
         public static final IOptionalNamedTag<Block> STONE = tag("stone");
         public static final IOptionalNamedTag<Block> STORAGE_BLOCKS = tag("storage_blocks");
+        public static final IOptionalNamedTag<Block> STORAGE_BLOCKS_AMETHYST = tag("storage_blocks/amethyst");
```

Comment:
Amethysts blocks aren't a storage block, they are crafted with 2x2 Amethysts and do not craft back into Amethyst

Reply:
Does not being able to craft amethyst blocks back into amethyst exclude it from being a storage block? Quartz blocks, for example, can also not be crafted back into quartz, yet it is a storage block for quartz.

Reply:
Hmm, this complicates things. I'd argue Quartz Blocks aren't storage blocks since they don't "store it" because you can't get it back. Storing it implies getting it back for later use.